### PR TITLE
openal: updated; patched new broken CMakeLists

### PR DESCRIPTION
**Pull request by PsychoXIVI**
;F

### CODE REVIEW

File: src/openal.mk [hunk 1]
```diff
@@ -18,7 +18,13 @@ define $(PKG)_UPDATE
 endef
 
 define $(PKG)_BUILD
+    # Update OpenAL cross compilation script with mxe root path
+    $(SED) -i "s|CMAKE_FIND_ROOT_PATH \"/usr/\$${HOST}\"|CMAKE_FIND_ROOT_PATH \"$(PREFIX)/$(TARGET)\"|" $(1)/XCompile.txt
+    cat $(1)/XCompile.txt
```
Comment (mabrand) [inline src/openal.mk]: stray line?

Comment (PsychoXIVI) [reply src/openal.mk]: Yes, line 23 is unnecessary. I forgot to remove it while testing I think.

Comment (mabrand) [reply src/openal.mk]: The package URL doesn't work for me. I found
    http://www.openal-soft.org/openal-releases/
The current version seems to be 1.19.1 Do you want to update the pull request for that?

**Comment by tonytheodore** [comment]
openal has since been updated

**Pull request closed without merging**